load.go

```diff
@@ -107,9 +108,28 @@ func (om *OpenMock) populateBehaviors(mocks []*Mock) {
 				r.AMQPMocks[m.Expect.AMQP] = append(r.AMQPMocks[m.Expect.AMQP], m)
 			}
 		}
+
+		if len(r.Behaviors[m.Key].Actions) > 0 {
+			orderedActions := r.Behaviors[m.Key].Actions
+			sort.Slice(orderedActions, func(i, j int) bool {
+				return orderedActions[i].Order < orderedActions[j].Order
+			})
+			if !actionsEqual(orderedActions, r.Behaviors[m.Key].Actions) {
+				m = r.Behaviors[m.Key].patchedWith(Mock{
+					Actions: orderedActions,
+				})
+				r.Behaviors[m.Key] = m
+			}
+		}
 	}
 }
 
+func actionsEqual(lhs []ActionDispatcher, rhs []ActionDispatcher) bool {
+	lhsString, lhsError := yaml.Marshal(lhs)
```

Comment:
does yaml marshal have consistent ordering for map type?

Reply:
I believe yes: https://blog.labix.org/2014/09/22/announcing-yaml-v2-for-go#mapslice